src/test/java/com/lists/LinkedListTest.java

```diff
@@ -0,0 +1,26 @@
+package com.lists;
+
+
+import junit.framework.TestCase;
+
+public class LinkedListTest extends TestCase {
+	
+	public void testList() {
+		
+		LinkedList<String> list = new LinkedList<String>();
+		list.add("quick");
+		list.add("brown");
+		list.add("fox");
+		list.add("jumped");
+		list.add("over");
+		list.add("little");
+		list.add("lazy");
+		list.add("dog");
+		
+		assertEquals(8,list.size());		
+		assertEquals(true,  list.remove(2));		
+		
+		
```

Comment:
include a test of an iterator, if you can. We have never tested Iterator ever. It could be a new addition for our DS.